diff --git a/templates/scopes.ldif b/templates/scopes.ldif
index 91ddbce..421b047 100644
--- a/templates/scopes.ldif
+++ b/templates/scopes.ldif
@@ -65,7 +65,7 @@ oxAuthClaim: inum=%(inumOrg)s!0005!36D9,ou=attributes,o=%(inumOrg)s,o=gluu
 oxAuthClaim: inum=%(inumOrg)s!0005!BE64,ou=attributes,o=%(inumOrg)s,o=gluu
 
 dn: inum=%(inumOrg)s!0009!6D97,ou=scopes,o=%(inumOrg)s,o=gluu
-defaultScope: true
+defaultScope: false
 description: Obtain UMA AAT
 displayName: http://docs.kantarainitiative.org/uma/scopes/authz.json
 inum: %(inumOrg)s!0009!6D97
@@ -73,7 +73,7 @@ objectClass: oxAuthCustomScope
 objectClass: top
 
 dn: inum=%(inumOrg)s!0009!6D96,ou=scopes,o=%(inumOrg)s,o=gluu
-defaultScope: true
+defaultScope: false
 description: Obtain UMA PAT
 displayName: http://docs.kantarainitiative.org/uma/scopes/prot.json
 inum: %(inumOrg)s!0009!6D96
diff --git a/templates/test/oxauth/data/oxauth-test-data.ldif b/templates/test/oxauth/data/oxauth-test-data.ldif
index 2f291a0..0d19164 100644
--- a/templates/test/oxauth/data/oxauth-test-data.ldif
+++ b/templates/test/oxauth/data/oxauth-test-data.ldif
@@ -314,19 +314,59 @@ oxAuthSkipAuthorization: true
 
 ######################################
 
+
 ######################################
 ######################################
 # UMA
 ######################################
+dn: inum=%(inumOrg)s!5214.40D1,ou=scopes,ou=uma,o=%(inumOrg)s,o=gluu
+objectClass: oxAuthUmaScopeDescription
+objectClass: top
+displayName: View
+inum: %(inumOrg)s!5214.40D1
+owner: inum=%(inumOrg)s!0000!A8F2.DE1E.D7FB,ou=people,o=%(inumOrg)s,o=gluu
+oxRevision: 0
+oxType: external
+oxUrl: http://photoz.example.com/dev/scopes/view
+
 dn: inum=%(inumOrg)s!B6EA.B301,ou=scopes,ou=uma,o=%(inumOrg)s,o=gluu
 objectClass: oxAuthUmaScopeDescription
 objectClass: top
 displayName: Add/Edit
 inum: %(inumOrg)s!B6EA.B301
-owner: inum=%(inumOrg)s!0000!A8F2.DE1E.D7FB,ou=peop
- le,o=%(inumOrg)s,o=gluu
+owner: inum=%(inumOrg)s!0000!A8F2.DE1E.D7FB,ou=people,o=%(inumOrg)s,o=gluu
 oxId: modify
 oxRevision: 0
 oxType: internal
 
+dn: inum=%(inumOrg)s!03F8.EDD7,ou=scopes,ou=uma,o=%(inumOrg)s,o=gluu
+objectClass: oxAuthUmaScopeDescription
+objectClass: top
+displayName: All
+inum: %(inumOrg)s!03F8.EDD7
+owner: inum=%(inumOrg)s!0000!A8F2.DE1E.D7FB,ou=people,o=%(inumOrg)s,o=gluu
+oxRevision: 0
+oxType: external
+oxUrl: http://photoz.example.com/dev/scopes/all
+
+dn: inum=%(inumOrg)s!EBD9.5EDC,ou=scopes,ou=uma,o=%(inumOrg)s,o=gluu
+objectClass: oxAuthUmaScopeDescription
+objectClass: top
+displayName: generate_id
+inum: %(inumOrg)s!EBD9.5EDC
+owner: inum=%(inumOrg)s!0000!A8F2.DE1E.D7FB,ou=people,o=%(inumOrg)s,o=gluu
+oxId: generate_id
+oxRevision: 0
+oxType: internal
+
+dn: inum=%(inumOrg)s!F366.715A,ou=resource_sets,ou=uma,o=%(inumOrg)s,o=gluu
+objectClass: oxAuthUmaResourceSet
+objectClass: top
+displayName: generate_id
+inum: %(inumOrg)s!F366.715A
+owner: inum=%(inumOrg)s!0000!A8F2.DE1E.D7FB,ou=people,o=%(inumOrg)s,o=gluu
+oxFaviconImage: http://example.com/generate_id_icon.png
+oxId: 1428339837931
+oxRevision: 1
+
 ######################################
